tests/Http/Profile/EditTest.php

```diff
@@ -126,7 +127,9 @@
 });
 
 test('email verification status is unchanged when the email address is unchanged', function () {
-    $user = User::factory()->create();
+    $user = User::factory()->create([
+        'email' => 'test@laravel.com',
```

Comment:
You can use fake()->freeEmail instead of fake()->email in the factory to make sure it generates valid email 🫡